agilent_infiniium_scope.py

```diff
@@ -31,446 +31,344 @@
-            if slope not in ['COMM','DEL','EDGE','GLIT','PATT','PWID','RUNT','SEQ',',SHOL','STAT','TIM','TRAN','TV',',WIND','SBUS1','SBUS2','SBUS3','SBUS4']:
+            if slope not in ['COMM', 'DEL', 'EDGE', 'GLIT', 'PATT', 'PWID',
+                             'RUNT', 'SEQ', 'SHOL', 'STAT', 'TIM', 'TRAN',
+                             'TV', 'WIND', 'SBUS1', 'SBUS2', 'SBUS3', 'SBUS4']:
                 raise Exception('Slope must be valid type.')
-            else:
-                yield dev.write('TRIG:MODE '+slope)
-                resp = yield dev.query('TRIG:MODE?')
+            yield dev.write('TRIG:MODE {}'.format(slope))
+        resp = yield dev.query('TRIG:MODE?')
         returnValue(resp)
 
-    @setting(133, slope = 's', returns = ['s'])
-    def trigger_edge_slope(self, c, slope = None):
+    @setting(133, slope='s', returns=['s'])
+    def trigger_edge_slope(self, c, slope=None):
         """Change trigger edge slope.
+
         Must be 'POS,' 'NEG', or 'EITH'er
         """
         dev = self.selectedDevice(c)
-        if slope is None:
-            resp = yield dev.query('TRIG:EDGE:SLOP?')
-        else:
+        if slope is not None:
             slope = slope.upper()
-            if slope not in ['POS','NEG','EITH']:
+            if slope not in ['POS', 'NEG', 'EITH']:
                 raise Exception('Slope must be "RISE" or "FALL"')
-            else:
-                yield dev.write('TRIG:EDGE:SLOP '+slope)
-                resp = yield dev.query('TRIG:EDGE:SLOP?')
+            yield dev.write('TRIG:EDGE:SLOP {}'.format(slope))
+        resp = yield dev.query('TRIG:EDGE:SLOP?')
         returnValue(resp)
 
-    @setting(134, mode = 's', returns = ['s'])
-    def trigger_sweep(self, c, mode = None):
+    @setting(134, mode='s', returns=['s'])
+    def trigger_sweep(self, c, mode=None):
         """Get or set the trigger mode
+
         Must be "AUTO", "TRIG" (normal), or "SING" (single)
         """
         dev = self.selectedDevice(c)
-        if mode is None:
-            resp = yield dev.query('TRIG:SWE?')
-        else:
+        if mode is not None:
             mode = mode.upper()
-            if mode not in ['AUTO','TRIG','SING']:
+            if mode not in ['AUTO', 'TRIG', 'SING']:
                 raise Exception('Mode must be "AUTO", "TRIG", or "SING".')
-            else:
-                yield dev.write('TRIG:SWE '+mode)
-                resp = yield dev.query('TRIG:SWE?')
+            yield dev.write('TRIG:SWE {}'.format(mode))
+        resp = yield dev.query('TRIG:SWE?')
         returnValue(resp)
 
-    @setting(150, side = 's', returns = ['s'])
-    def horiz_refpoint(self, c, side = None):
-        """Get or set the reference point for the horizontal position. Must be 'LEFT', 'CENT'er, or 'RIGH't.
+    @setting(150, side='s', returns=['s'])
+    def horiz_refpoint(self, c, side=None):
+        """Get or set the reference point for the horizontal position.
+
+        Must be 'LEFT', 'CENT'er, or 'RIGH't.
```

Comment:
This might be confusing to people. I would just say `'LEFT', 'CENT', or 'RIGH'`.